mlflow/server/handlers.py

```diff
@@ -144,6 +145,20 @@ def _get_artifact_repo_mlflow_artifacts():
     return _artifact_repo
 
 
+def _get_serve_artifacts_mode():
+    """
+    Set whether the MLflow server is operating in ``--serve-artifacts`` mode for artifact listing
+    endpoint resolution.
```

Comment:
```suggestion
    Initializes `_serve_artifacts_mode` that indicates whether the MLflow server is operating in ``--serve-artifacts`` mode for artifact listing, and returns it.
    endpoint resolution.
```

Nit: can we fix the docstring to indicate this function returns something?

Reply:
changed